mitodl/mit-open
# Add marketing images to homepage testimonial, fix some styling issues

### What are the relevant tickets?

Closes mitodl/hq#4559 
Closes mitodl/hq#4541

### Description (What does it do?)

1. Adds the marketing images to the static folder, per the issue.
2. Updates the cards to use a random image from that list rather than the avatar or cover image.
3. Adds the avatar image to the name block on the card
4. Styling fixes for the quote, name block, and quote leader
5. Change the button style to match the other pager buttons
6. Add tests for testimonial sliders

### Screenshots (if appropriate):

Regular desktop:
![Screenshot 2024-06-17 at 10 20 55 AM](https://github.com/mitodl/mit-open/assets/945611/b62af164-4e8e-46a6-a97b-46b4141763da)

Extra-wide desktop:
![Screenshot 2024-06-17 at 10 21 04 AM](https://github.com/mitodl/mit-open/assets/945611/3d40e958-4fa1-4ab9-a63b-ab0159fe84d6)

Mobile:
![Screenshot 2024-06-17 at 10 21 17 AM](https://github.com/mitodl/mit-open/assets/945611/fddf345a-e48f-4747-ba93-fc5a6ce8fec1)

### How can this be tested?

Automated tests should pass. This includes some new tests for the testimonial sliders, so those specifically should pass - they're in `HomePage.test.tsx` and `TestimonialDisplay.test.tsx`.

The layout should more closely match what's in Figma now, and the cover image on the left should be one of the six marketing images. This just chooses one at random, it makes no effort to check to see if one's already been chosen or not.

For the unit page testimonials: the attestant block is now its own component - this should have no real change on the unit page sliders but it is a change.
